test/hoya_iranian/visual_config/hoya.json

```diff
@@ -0,0 +1,87 @@
+{
+    "model": "Hoya",

```

Comment:
"model": "Hoya_iranian"
Same for file (in that way we can distinguish them easily)